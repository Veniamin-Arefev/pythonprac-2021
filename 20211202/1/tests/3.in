class C(metaclass=dump):
    def __init__(self, string):
        self.val = string

    def __add__(self, other):
        return (self.val + other.val)


a, b = C("plus"), C("minus")
print(a + b)