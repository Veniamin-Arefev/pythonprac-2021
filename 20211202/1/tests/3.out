__init__: ('plus',), {}
__init__: ('minus',), {}
__repr__: (), {}
__add__: (minus,), {}
plusminus
